src/platform/user/profile/vet360/tests/actions/transactions.unit.spec.js

```diff
@@ -0,0 +1,81 @@
+import {
+  validateAddress,
+  ADDRESS_VALIDATION_CONFIRM,
+} from '../../actions/transactions';
+import sinon from 'sinon';
+import { expect } from 'chai';
+
+const route = 'foo';
+const method = 'PUT';
+const fieldName = { value: 'mailingAddress' };
+const payload = {
+  id: 123,
+  addressLine1: '1493 Martin Luther King Rd',
+  addressLine2: 'string',
+  addressLine3: 'string',
+  addressType: 'DOMESTIC',
+  city: 'Fulton',
+  countryName: 'United States',
+  stateCode: 'NY',
+  zipCode: '97062',
+  addressPou: 'CORRESPONDENCE',
+};
+const analyticsSectionName = 'bar';
+
+describe('validateAddress', () => {
+  it('dispatches ...', done => {
+    const dispatch = sinon.spy();
+
+    const forceMockResponse = true;
+
+    const thunk = validateAddress(
+      route,
+      method,
+      fieldName,
+      payload,
+      analyticsSectionName,
+      forceMockResponse,
```

Comment:
Innnnnnteresting and creative! In the past when we test action creators that make network requests is so mock the fetch just for a particular request, which also allows us to mock failed requests. These tests show a few different ways to mocking fetch locally with our `mockFetch` and `mockApiRequest` helpers: https://github.com/department-of-veterans-affairs/vets-website/blob/630d391459f17dc8e3e6cc6a464582d3d7442e9d/src/applications/hca/tests/actions.unit.spec.js#L36-L131. I _prefer_ doing it by mocking fetch in the test file so we aren't adding logic in the action creator to play nicely with our tests. Adding this `forceMockResponse` arg works in this case because the action creator does the `isVet360Configured()` check and uses the `localVet360` mocks, but that's fairly irregular for our code most of the time.